routes/productRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const productController = require("../controllers/productController.js");
+
+const router = express.Router();
+
+router.get("/list", productController.allProducts);
```

Comment:
RESTful API를 설계하는 데 있어, 이미 index.js 파일에서 `/products`라는 리소를 명시하고 있기 때문에 `list`라는 리소스를 명시하지 않아도 될 것 같습니다.
```suggestion
router.get("/", productController.allProducts);
```